develop-docs/development-infrastructure/devservices.mdx

```diff
@@ -84,23 +84,65 @@ devservices up --mode symbolicator
 
 ## Running a dependency locally
 
-If you want to run a dependency locally rather than as a container, you can do so by toggling the runtime to LOCAL.
+You can run a dependency locally instead of as a container by toggling its runtime via the `devservices toggle` command.
```

Comment:
```suggestion
You can run a dependency locally by toggling its runtime via the `devservices toggle` command.
```

Reply:
I personally think it's better to be explicit here about the fact that local implies not running via a container